fixBindgenShim.js

```diff
@@ -114,12 +114,14 @@ export const setRandomSeed = (seed) => {\n\
     //prettier-ignore
     const getRandomValuesReplacementCode =
     `const randBuffer = getArrayU8FromWasm(arg1, arg2);
-     if(randomSeed instanceof Uint8Array && randomSeed.length === 32) {\n\
-       randBuffer.set(randomSeed, 0);\n\
-       randomSeed = undefined;\n\
-     } else {\n\
-       getObject(arg0).getRandomValues(randBuffer);\n\
-     }\n\
+    if(getObject(arg0) && typeof getObject(arg0).getRandomValues === "function") {
```

Comment:
What is this new branch doing?

Reply:
Checking if we have the ability to use WebCrypto to generate random values. Instead of always first using the seed if it's set, we first try to use an actual source of known random values and only fall back to the seed if we need to.